deps.edn

```diff
@@ -1,4 +1,4 @@
-{:paths ["resources" "src"]
+{:paths ["src"]
```

Comment:
So this is a problem I ran into last week @otfrom. witan.phyrexian requires the `resources` dir, and also having `resources` here seems to override the location of `resources` when trying to call witan.phyrexian from another library. Just taking it out of here probably isn't the whole solution, but it resolved it for now, and witan.send.vis doesn't requires access to `witan.send.vis/resources` currently anyway, so that doesn't cause an issue. I'm fairly confident that this is a problem partially because I'm calling most of these libraries from `:root/local` in the deps